filebeat/module/haproxy/http/manifest.yml

```diff
@@ -0,0 +1,13 @@
+module_version: 1.0
+
+var:
+  - name: paths
+    default:
+      - /var/log/haproxy.log
+  - name: syslog_port
+    default: 9001
+  - name: input
+    default: syslog
```

Comment:
Being syslog the default is going to be tricky for the `test_modules` test as it instantiates filebeat with the default configurations.
@kvch @ph is there a way to provide a different config than the default for the `test_modules`?

Reply:
`filebeat/tests/system/config/filebeat.yml.j2` is the template used during running system tests. This needs to be extended with syslog input.

Reply:
The current system test framework also needs to be extended to be able to send logs over TCP and UDP to Filebeat. Maybe running netcat with proper config does the trick of feeding Filebeat with input.

Reply:
Thanks, I'll take a look to this.